pkgs/development/compilers/symbiflow-yosys-plugins/make.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/fasm-plugin/Makefile b/fasm-plugin/Makefile
+index dc171a4..c050fbb 100644
+--- a/fasm-plugin/Makefile
++++ b/fasm-plugin/Makefile
+@@ -2,7 +2,7 @@ CXX = $(shell yosys-config --cxx)
+ CXXFLAGS = $(shell yosys-config --cxxflags)
+ LDFLAGS = $(shell yosys-config --ldflags)
+ LDLIBS = $(shell yosys-config --ldlibs)
+-PLUGINS_DIR = $(shell yosys-config --datdir)/plugins
++PLUGINS_DIR ?= /usr/share/yosys/plugins
```

Comment:
Makes sense to submit the `s/ = / ?= /` change to the upstream.

Reply:
I suppose it's not overrideable via `makeFlags = [ ];` because its a recursive make?